src/System.IO/tests/Stream/NullTests.cs

```diff
@@ -0,0 +1,104 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace StreamTests
+{
+    public class NullCtorTests
+    {
+        [Fact]
+        public static void TestNullStream()
+        {
+            var t = TestNullStream(Stream.Null);
+            t.Wait();
```

Comment:
It'd be better to do this as:
```C#
[Fact]
public static Task TestNullStream()
{
    return TestNullStream(Stream.Null);
}
```
or
```C#
[Fact]
public static async Task TestNullStream()
{
    await TestNullStream(Stream.Null);
}
```
The way it's currently written could lead to deadlock problems.